megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -4274,12 +4274,15 @@ else if ((atype != null)
         boolean mekMortarMunitionsIgnoreImmobile = wtype != null && wtype.hasFlag(WeaponType.F_MEK_MORTAR) 
                 && (atype != null) && (munition == AmmoType.M_AIRBURST);
         if (wtype != null && !(wtype instanceof ArtilleryCannonWeapon) && !mekMortarMunitionsIgnoreImmobile) {
-            ToHitData immobileMod = Compute.getImmobileMod(target, aimingAt, aimingMode);
+            ToHitData immobileMod = null;
```

Comment:
The null assignment isn't required given it will be initialized immediately after.